### ISSUE TITLE
Search within a single title

### ISSUE THREAD

**Issue by sseymore** [uoregon-libraries/oregon-oni#68]
I realized I never ticketed this patron request..

TL;DR: Users want a better way to search within a specific title without using the advanced search.

Use case explanation:
"We anticipate that most the those interested in our student newspaper will be former students and members of our campus community.  Rather than doing in depth historical research, they would most likely be interested in a person or campus event.  We would like to make sure that searching The Beacon would be straightforward.  In looking at the public interface for Historic Oregon Newspaper, I could find direct links to descriptive information about individual titles but not a direct link to search a single title.  From what I understood, the only way to search an individual title is through the advanced search.   Is this correct?   We think this might be a barrier for some of our users.     If this is the case, are there any plans to enhance search capabilities to more easily limit to a single newspaper?"

They decided not to pursue digitization/a grant with us because of this reason.
